src/design-system/components/Toggle/Toggle.tsx

```diff
@@ -38,8 +44,12 @@ const Toggle = ({
         uncheckedIcon={false}
         checkedIcon={false}
         onColor={accentColor || avatar?.color || '#268FFF'}
-        activeBoxShadow={accentColor || avatar?.color || '#268FFF'}
+        boxShadow={`0px 0px 1px 2px transparent`}
+        activeBoxShadow={`0px 0px 2px 4px ${
+          accentColor || avatar?.color || '#268FFF'
+        }`}
```

Comment:
original code was broken, added transparent box shadow because it behaves oddly without it.